Lib/fontTools/feaLib/variableScalar.py

```diff
@@ -109,3 +115,88 @@ def add_to_variation_store(self, store_builder):
         store_builder.setSupports(supports)
         index = store_builder.storeDeltas(deltas)
         return int(self.default), index
+
+    def __sub__(self, other):
+        if not isinstance(other, (VariableScalar, float, int)):
+            return NotImplemented
+        new = VariableScalar()
+        new.axes = self.axes
```

Comment:
curious why VariableScalar's default constructor doesn't take an `axes` parameter but you set it after the fact?